libs/providers/flagd/src/lib/service/grpc/service.ts

```diff
@@ -206,17 +206,37 @@ export class GRPCService implements Service {
   private objectParser = (struct: Struct) => {
     if (struct !== undefined) {
```

Comment:
maybe it is worth to also nullcheck here
```suggestion
    if (struct) {
```